### PR TITLE
wordpressPackages: update and add license information

### LINKED ISSUE #202191: Missing licenses in wordpressPackages set
## Issue description

The package with `wordpressPackages` that are automatically fetched carry no license.

### Steps to reproduce

````
nix-repl> wordpressPackages.plugins.gutenberg.meta.license
error: attribute 'license' missing

       at «string»:1:1:

            1| wordpressPackages.plugins.gutenberg.meta.license
             | ^
````

cc @onny

### PR DESCRIPTION
**Pull request by onny**
## Description of changes

Fixes https://github.com/NixOS/nixpkgs/issues/202191

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Atemu** [comment]
Could you provide a bit of background how these were determined? Especially only vs. plus.

**Comment by onny** [comment]
> Could you provide a bit of background how these were determined? Especially only vs. plus.

i looked them all up by hand, for example https://plugins.svn.wordpress.org/wpforms-lite/trunk/readme.txt

**Comment by Atemu** [comment]
I see. I had assumed these were auto-generated because they're inside a JSON. If the values require human input anyways, is there any reason to keep this a JSON rather than a Nix expression? As demonstrated, it can be confusing to an outsider ;)

**Comment by onny** [comment]
the nix expression gets generated by the json input file. this is the same approach already used in the nextcloud package set (nc4nix)

File: pkgs/servers/web-apps/wordpress/packages/plugins.json [hunk 1]
```diff
@@ -6,10 +6,10 @@
...
   },
   "akismet": {
-    "path": "akismet/tags/5.3",
-    "rev": "2991242",
-    "sha256": "1i7y09x2aikndf80wl7xv65m71nv5s0w6jcadpp0dsyzp2m3gjnn",
-    "version": "5.3"
+    "path": "akismet/tags/5.3.1",
+    "rev": "3023195",
+    "sha256": "0ma1d7s535bcjs8x7shzx0qxhxm4cw028rf55l26pkxpw5s7mfvv",
+    "version": "5.3.1"
```
Comment (Atemu) [inline pkgs/servers/web-apps/wordpress/packages/plugins.json:12]: What's up with all of these lockfile changes? Is this an update? If so, that should happen in a separate commit and perhaps even a separate PR.

**Review by Atemu — COMMENTED** [review]
I see, the update script also uses the JSONs.

One final quesiton, the rest LGTM.

**Comment by onny** [comment]
sorry this was a mistake :(
